src/services/s3Submit.ts

```diff
@@ -141,15 +148,18 @@ const uploadToS3 = <T>(
 }
 
 const uploadFormSubmission = (
-  s3Configuration: SubmissionTypes.S3UploadCredentials,
+  s3Configuration: {
+    credentials: SubmissionTypes.S3UploadCredentials['credentials']
+    s3: SubmissionTypes.S3UploadCredentials['s3']
+  },
   formJson: {
     definition: FormTypes.Form
-    submission: FormTypes.FormSubmission['submission']
+    submission: SubmissionTypes.FormSubmission['submission']
     submissionTimestamp: string
     keyId?: string
     formsAppId: number
   },
-  tags: Record<string, string>,
+  tags: UnknownObject,
```

Comment:
This is not `<string, unknown>`, its `<string, string>`.

Reply:
The jobId can be `null | undefined` as well :/